cmd/ci-operator/main.go

```diff
@@ -627,6 +633,18 @@ func (o *options) Complete() error {
 			return fmt.Errorf("could not get upload secret %s from path %s: %w", api.GCSUploadCredentialsSecret, o.uploadSecretPath, err)
 		}
 	}
+
+	if o.hiveKubeconfigPath != "" {
+		kubeConfigs, _, err := util.LoadKubeConfigs(o.hiveKubeconfigPath, nil)
+		if err != nil {
+			return fmt.Errorf("could not load kube config from path %s: %w", o.hiveKubeconfigPath, err)
+		}
+		kubeConfig, ok := kubeConfigs[string(api.ClusterAPPCI)]
```

Comment:
I would expect it to just always contain exactly one config. This might not be app.ci in the future

Reply:
Updated.